netanalysis/ip/ip_info_test.py

```diff
@@ -4,54 +4,71 @@
 
 from . import ip_info as ii
 
+localhost_ip4 = ip("127.0.0.1")
```

Comment:
Please revert the localhost IPs. Having the IPs explicitly is more readable, since they are well known. (you don't have ZERO = 0, for example)